src/lsm/compaction.zig

```diff
@@ -223,8 +234,11 @@ pub fn CompactionType(
                 // TODO(Compaction Pacing): Reserve smaller increments, at the start of each beat.
                 // (And likewise release the reservation at the end of each beat, instead of at the
                 // end of each half-bar).
-                // TODO(Move Table) Don't reserve these when we just move the table to the next level.
-                .grid_reservation = grid.reserve(range.table_count * Table.block_count_max).?,
+                .grid_reservation = switch (strategy) {
+                    // Don't reserve these when we just move the table to the next level.
+                    .move => null,
+                    .merge => grid.reserve(range.table_count * Table.block_count_max).?,
```

Comment:
Let's use consistent order everywhere, also to follow the typical logical order of use (and the reason we re-ordered the enum): `merge` always before `move`.